devfiles/go-web-sample/meta.yaml

```
@@ -0,0 +1,6 @@
+---
+displayName: Go Real World Application
+description: Go Stack with Go 1.12.9 and Real Wold Application
```

just to inform there is a typo on WoRld
```suggestion
description: Go Stack with Go 1.12.9 and Real World Application
```